ChangeLog

```diff
@@ -9,6 +9,9 @@ Release date: Undefined
 ..
   Put new features here and also in 'doc/whatsnew/2.8.rst'
 
+* Apply ``const-naming-style`` to module constants annotated with
+  ``typing.Final``
```

Comment:
Should it be a `2.8` or `2.7.5` feature?

Reply:
Depends on the end result of the discussion in #4263 :)